### DISCUSSION TITLE
Get information about the frames in animated webp

### DISCUSSION THREAD

**Discussion by avkabanov** [ImageMagick/ImageMagick#2837] [Help]
I have animated WebP as the input, next I want to process the image using ImageIO. Since I can not use ImageIO for animated images, I want to split the input image into frames, make some processing on each frame, and merge all frames into a resulting WebP.

The thing is, in order to merge them into a single animated image, I need to have information about the frames, like frame duration and loop count. How can I get it? 
I've tried to extract profiles using `webpmux` but it gave me `ERROR (WEBP_MUX_NOT_FOUND): Could not get the EXIF metadata.
`

Maybe there is another way to execute processing with ImageIO on animated WebP?

**Comment by snibgo** [comment]
What is ImageIO? This is an ImageMagick ("IM") forum.

With IM, you can extract frames, and get metadata such as frame delay.

```
magick myfile.webp info:

magick myfile.webp -verbose info: |grep Delay:
```

**Reply by avkabanov** [reply]
Thank you! I think that's exactly what I need! Just a quick question: I see a delay as `10x100`. Does that mean that the delay is 1000 or 10 and 100 determines different values?

**Comment by snibgo** [comment]
"Delay: 10x100" means there are 10 units (or "ticks") of delay, and each unit is 1/100 of a second. So the delay in seconds is 10/100 = 0.1 seconds.
